db/flush_job.h

```diff
@@ -131,6 +135,10 @@ class FlushJob {
   // In this case, only after all flush jobs succeed in flush can RocksDB
   // commit to the MANIFEST.
   const bool write_manifest_;
+  // The current flush job can commit flush result of a concurrent flush job.
+  // We collect FlushJobInfo of all jobs committed by current job and fire
+  // OnFlushCompleted for them.
+  autovector<FlushJobInfo*> committed_flush_jobs_info_;
```

Comment:
I should have make it hold unique_ptr of `FlushJobInfo`, but got compile error since unique_ptr cannot be copy-construct.

Reply:
Fixed